src/deserialize/mod.rs

```diff
@@ -9,6 +13,93 @@ use std::{
     str::FromStr,
 };
 
+#[cfg(feature = "geo-types")]
+pub mod geo_types;
+
+/// Deserializes any type which implements TryFromWkt from a WKT string.
```

Comment:
What do you think about including a sentence that clarifies that this functionality is intended for working with structured data (I know this is directly implied by the use of `serde`, but that might not be obvious to the beginner or casual Rust user) as opposed to WKT strings?

Reply:
Good point. I've tried to rework the docs to be a bit clearer, as well as re-ordered the doc examples in https://github.com/georust/wkt/pull/99/commits/5129bb9ac336471da8e173a63110a023f94d2f38

I'll merge now, but let me know if you'd like further changes.

Reply:
That was the only thing that leaped out at me, and the updated docs look good!